core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2599,6 +2599,33 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     testAppendConfig(props, "0:0", "1:1,0:0")
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("quorum=kraft"))
+  def testListClientMetricsResources(ignored: String): Unit = {
```

Comment:
it must be `quorum`. see https://github.com/apache/kafka/blob/7c539f902983a76f14d0cd993e7d6dcbfdacd909/core/src/test/scala/kafka/utils/TestInfoUtils.scala#L38

Reply:
This type of writing places the parameter value in the final display name. so we will run with kraft

Reply:
ok, that can work.